sentry_sdk/integrations/anthropic.py

```diff
@@ -165,11 +166,16 @@ def _sentry_patched_create_common(f, *args, **kwargs):
         span.set_data(SPANDATA.AI_STREAMING, False)
 
         if should_send_default_pii() and integration.include_prompts:
-            span.set_data(SPANDATA.AI_INPUT_MESSAGES, messages)
+            span.set_data(
+                SPANDATA.AI_INPUT_MESSAGES, _serialize_span_attribute(messages)
```

Comment:
Should we maybe just call `_serialize_span_attribute` directly in `POTelSpan.set_attribute`? (which `set_data` calls under the hood if I'm not mistaken)

Reply:
Good idea! Will create a PR for that!